docs/faststore/docs/customization/building-sections/override-use-cases/adding-an-image-to-the-alert-component.mdx

```diff
@@ -0,0 +1,174 @@
+
+By leveraging `getOverriddenSection`, you can create a custom component called `AlertWithImage` that achieves the desired functionality. Follow the steps below:
+
+1. In the `src/components/sections` folder, create the folder `AlertWithImage` folder.
+2. In the `AlertWithImage` folder, create the `AlertWithImage.tsx` file.
+3. Import the following to the `AlertWithImage.tsx` file:
+
+   ```tsx
+   import { getOverriddenSection } from "@faststore/core";
+   import Image from "next/image";
+   import { useMemo } from "react";
+   ```
+
+4. Define the `AlertWithImageProps` interface:
+
+   ```tsx focus=5:9
+   import { getOverriddenSection } from "@faststore/core";
+   import Image from "next/image";
+   import { useMemo } from "react";
+
+   interface AlertWithImageProps
+     extends Omit<React.ComponentProps<typeof AlertSection>, "icon"> {
+     src: string;
+     alt: string;
+   }
+   ```
+
+   - This interface defines the props accepted by the `AlertWithImage` component.
+   - It inherits all props from the native Alert component, excluding the icon prop, which we intend to replace with an image.
+   - It adds a new prop named `src`, which specifies the image source URL.
+
+5. Create the `AlertWithImage` component:
+
+   ```tsx focus=10:31
+   import { useMemo } from "react";
+   import { AlertSection, getOverriddenSection } from "@faststore/core";
+   import { Image_unstable as Image } from "@faststore/core/experimental";
+
+   interface AlertWithImageProps
+     extends Omit<React.ComponentProps<typeof AlertSection>, "icon"> {
+     src: string;
+     alt: string;
+   }
+
+   export default function AlertWithImage(props: AlertWithImageProps) {
+     const { src, alt, ...otherProps } = props;
+
+     const OverriddenAlert = useMemo(
+       () =>
+         getOverriddenSection({
+           Section: AlertSection,
+           className: styles.alertWithImage,
+           components: {
+             Icon: {
+               Component: () => (
+                 <Image
+                   src={props.src}
+                   alt={props.alt}
+                   width={24}
+                   height={24}
+                 />
+               ),
+             },
+           },
+         }),
+       []
+     );
+
+     return <OverriddenAlert {...otherProps} icon="" />;
+   }
+   ```
+
+   - This function receives the `AlertWithImageProps`.
+   - It destructures the `src` prop containing the image source and remaining props using the spread operator (`...otherProps`).
+   - The core functionality lies within the `useMemo` hook.
+
+     - It defines the `OverriddenAlert` constant using `getOverriddenSection`.
+     - The override options specify:
+       - **`Section`:** The section to be overridden (Alert).
+       - **`components`:** An object containing the override for the Icon component.
+
+   - Inside the Icon override:
+
+     - **`Component`:** A function returning an `Image` component. This displays the image specified by the `src` prop.
```

Comment:
It would be great to link this Image Component [doc](https://github.com/vtexdocs/dev-portal-content/pull/1264/files) here, I couldn't find it in the portal.